webpack/ForemanInventoryUpload/Components/PageHeader/components/PageDescription/PageDescription.js

```diff
@@ -53,6 +53,16 @@ export const PageDescription = () => {
           />
         </Text>
       )}
+      <Text ouiaId="text-minimal-data-collection">
+        {__('Learn more about ')}
+        <a
+          href="https://content-preview.docs.redhat.com/en/documentation/red_hat_satellite/6.18/html-single/managing_hosts/index#setting-minimal-data-collection"
```

Comment:
This link points to 6.18 doc. 
Do we update the link when the doc got updated about minimal data collection or for each new release?